dizoo/trading_test/config/calculate_max_profit.py

```diff
@@ -0,0 +1,9 @@
+from easydict import EasyDict
```

Comment:
use pytest to implement unittest, you can get started from this [guideline](https://di-engine-docs.readthedocs.io/zh_CN/latest/22_test/index_zh.html)